content/tutorials/Super-lightweight-realtime-subscriptions-with-SSE-and-Ably.textile

```diff
@@ -0,0 +1,223 @@
+---
+title: Super-lightweight realtime subscriptions with SSE and Ably
+section: tutorials
+index: 109
+languages:
+  - javascript
+  - python
+  - php
+---
+
+h2(#what-is-sse). What is SSE?
+
+Server-sent events is a standard describing how servers can initiate data transmission towards clients once an initial client connection has been established. They are commonly used to send message updates or continuous data streams to a browser client and designed to enhance native, cross&#45;browser streaming through a JavaScript API called EventSource, through which a client requests a particular URL in order to receive an event stream. Basically, a server&#45;sent event is when a web page automatically gets updates from a server.
+
+h2(#setting-up-the-ui-and-publisher). Setting up the UI and publisher
+
+*Setting up the UI*
+
+Let us create a simple HTML file that implements SSE using Ably. Create a file called @example.html@ and add:
+
+```[html]
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+        <style>
+          ul {
+              width: 60%;
+              margin-top: 10px;
+              margin: 0 auto;
+              font-family: courier, courier new;
+              background-color: #333;
+              height: 300px;
+              overflow: scroll;
+              color: orange;
+          }
+          h1{
+            text-align: center;
+          }
+          h3{
+            text-align: center;
+          }
+    </style>
+  </head>
+  <body>
+    <h1>SSE example</h1>
+
+    <h3>Events:</h3>
+    <ul id="events"></ul>
+  </body>
+</html
+```
+
+What's happening in the code above? We create a basic HTML structure which consists of an @ul@ element. The children of this element will be dynamically populated.
+
+*Setting up the Publisher* The Publisher refers to the server-side aspect, where the messages are being sent. For the purpose of this tutorial, we will use a simple example that sends the current date and time as Ably events every two seconds. Here is what our code looks like:
+
+
+blang[javascript].
```

Comment:
Samuelo, you've only included different languages on the publisher side, we'll need to show examples in the subscriber side as well, in fact, that is more important as people may want to use SSE to subscribe to data without an SDK for example. In such a case, you'll need to have the index.html page also as language specific